DB/src/main/java/io/deephaven/db/v2/IndexedDataColumn.java

```diff
@@ -156,14 +157,9 @@ public byte getByte(final long pos) {
     public byte[] getBytes(final long startPosInclusive, final long endPosExclusive) {
         final Index rangeIndex = getSubIndexByPos(startPosInclusive, endPosExclusive);
```

Comment:
I just noticed this while reviewing: `rangeIndex` should be closed when we're done with it. Older code isn't so careful, but we can keep improving as we go. If you add it to an enclosing try-with-resources, or refactor to get it into the existing one, that will do the job.
```
        try (final Index rangeIndex = getSubIndexByPos(startPosInclusive, endPosExclusive);
             final ChunkSource.FillContext context = columnSource.makeFillContext(rangeIndex.intSize("getBytes"), null)) {
            final byte[] result = new byte[rangeIndex.intSize("getBytes")];
            columnSource.fillChunk(context, WritableByteChunk.writableChunkWrap(result), rangeIndex);
        }
```

Reply:
added 'finally' block to close rangeIndex